Utilities/DataLoading/EdFi.LoadTools/ApiClient/TokenRetriever.cs

```diff
@@ -38,17 +38,21 @@ public TokenRetriever(IOAuthTokenConfiguration configuration)
 
         public async Task<BearerToken> ObtainNewBearerToken()
         {
+            var requestMessage = new HttpRequestMessage
+            {
+                Method = HttpMethod.Post,
+                Content = new StringContent("Grant_type=client_credentials", Encoding.UTF8, "application/x-www-form-urlencoded"),
+                RequestUri = new Uri(_configuration.Url)
+            };
+
```

Comment:
I am was a little concerned about this change, however after researching, I am more comfortable with the change. The reason for the change is to reset the header values and force them to change. c.f. https://stebet.net/httpclient-best-practices/